components/style/properties/gecko.mako.rs

```diff
@@ -1362,7 +1365,16 @@ ${impl_style_struct(style_struct)}
 ${define_ffi_struct_accessor(style_struct)}
 % endfor
 
-static mut INITIAL_GECKO_VALUES: *mut ComputedValues = 0 as *mut ComputedValues;
+// To avoid UB, we store the initial values as a atomic. It would be nice to
+// store them as AtomicPtr, but we can't have static AtomicPtr without const
+// fns, which aren't in stable Rust.
+static INITIAL_VALUES_STORAGE: AtomicUsize = ATOMIC_USIZE_INIT;
+unsafe fn raw_initial_values() -> *mut ComputedValues {
```

Comment:
I still haven't found any reference to undefined behavior for doing what we were doing, but anyway, I've verified that using `AtomicPtr` with `Ordering::Relaxed` is only 1 instruction more in x86.

@bors-servo: r+